nni/mutable/mutable.py

```diff
@@ -678,6 +699,12 @@ def freeze(self, sample: Sample) -> Any:
     def __len__(self):
         return len(self.values)
 
+    def as_legacy_dict(self) -> dict:
+        return {
+            '_type': 'choice',
+            '_value': self.values,
```

Comment:
Suggest to add type hint for fields. It took me some time to find out `self.values`' type.

Reply:
Are you suggesting `TypedDict`?

I don't want to waste the effort because it's **legacy**_dict.

Reply:
I mean `self.values: list[Any] = values`

Reply:
My IDE inferred these types automatically. But I can surely add them on.